packages/FileUpload/src/FileList/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import DocumentIcon from "../DocumentIcon";
+
+const IMAGE = "image";
+
+const FileList = ({
+  previewImages,
+  handleCancel
+}: {
+  previewImages: any;
+  handleCancel: any;
+}) => (
+  <>
+    {previewImages.map((file: any) => (
```

Comment:
can destructure { type, base64, name, id}

Reply:
I prefer don't do that because I will have to do default params also, because base64 is not coming always.